### DISCUSSION TITLE
SourceCodeExpander UI improvement

### DISCUSSION THREAD

**Discussion by Seledy219** [Strypper/mauisland#24] [Ideas]
![image](https://user-images.githubusercontent.com/121172956/211271341-a9ebbefe-78e4-4a1f-bf09-dfa777925fbd.png)
![image](https://user-images.githubusercontent.com/121172956/211271436-ccd2bffb-60db-43fc-a557-3e19e730d0f0.png)

**Comment by Strypper** [comment]
Cool feature I will have a look at this to improve our existing SourceCodeExpander
